packages/app/src/SearchInputV2.tsx

```diff
@@ -17,6 +17,7 @@ export default function SearchInputV2({
   connectionId,
   enableHotkey,
   onSubmit,
+  additionalSuggestions,
```

Comment:
I think this works for now, though I wonder if the v1 pattern of having a separate metrics tag filter input would be cleaner? We can leave it as a follow up.

https://github.com/hyperdxio/hyperdx/blob/main/packages/app/src/MetricTagFilterInput.tsx

Reply:
I like that idea. Will add a ticket to follow up on this and the other suggestions. Thanks!

Reply:
New ticket created: HDX-1410